src/dug/core/parsers/dbgap_parser.py

```diff
@@ -14,6 +15,13 @@ class DbGaPParser(FileParser):
 
     @staticmethod
     def parse_study_name_from_filename(filename: str):
+        stemname = os.path.splitext( os.path.basename(filename) )[0]
```

Comment:
When dealing with filesystem paths, you might find python's [pathlib](https://docs.python.org/3/library/pathlib.html) helpful